### PR TITLE
Add trasition transform to book list

### PR DESCRIPTION
**Pull request by blacksam07**
## Summary
- Add transition transform to item list

## Trello Card

[35-transitions](https://trello.com/c/TuuaSnSD/35-transitions)

## Screenshot

transition-list

### CODE REVIEW

File: scss/index.scss [hunk 1]
```diff
@@ -18,6 +18,11 @@
   width: 100%;
 }
 
+.list-item:hover {
```
Comment (CrisVega08) [inline scss/index.scss]: Use sass Nesting

File: scss/index.scss [hunk 2]
```diff
@@ -18,6 +18,11 @@
...
+  transform: scale(1.05);
+  transition: transform 0.5s;
```
Comment (CrisVega08) [inline scss/index.scss]: Define transition on .list-item class

File: scss/index.scss (hunk 1 above)
Comment (CrisVega08) [inline scss/index.scss]: Also remember use :: for pseudo elements

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/index.scss [hunk 3]
```diff
@@ -15,9 +15,15 @@
   margin: 15px 15px;
```
Comment (betoobernal) [inline scss/index.scss]: assing unary value will be enough. Only 15px. 😅

**Review by CrisVega08 — APPROVED** [review]

**Review by betoobernal — APPROVED** [review]

**Pull request merged**